src/simulations/paramscan.jl

```diff
@@ -15,20 +15,18 @@ Keys of `parameters` should be of type `Symbol`.
 of which all values in `parameters` should be a subset. This means `parameters`
 can take both model and agent constructor properties.
 
-### Keywords
-All the following keywords are propagated into [`run!`](@ref).
-Defaults are also listed for convenience:
-`agent_step! = dummystep, n = 1, when = 1:n, model_step! = dummystep`,
-`step0::Bool = true`, `parallel::Bool = false`, `replicates::Int = 0`.
-Keyword arguments such as `adata` and `mdata` are
-also propagated.
-
+## Keywords
 The following keywords modify the `paramscan` function:
 
-`include_constants::Bool=false` determines whether constant parameters should be
-included in the output `DataFrame`.
+- `include_constants::Bool=false` determines whether constant parameters should be
+  included in the output `DataFrame`.
```

Comment:
Not directly related to this PR but maybe it might be nice to be able to pass a list of parameters that should be included. This way users could pick which constants should be included and which should be left out of the df.

Reply:
You can simply not include them in `parameters` though, is this good enough?

Reply:
I don't think I understand what you mean by this. If I normally provide some constant model properties like the grid size within `parameters`, then I cannot just leave them out of `parameters` because they're crucial for the model. What do you mean by saying "simply not include them in `parameters`"?